htdocs/product/list.php

```diff
@@ -1972,7 +1972,9 @@
 					if ($product_fourn->product_fourn_price_id > 0) {
 						if ((isModEnabled("fournisseur") && $user->hasRight('fournisseur', 'lire') && !getDolGlobalString('MAIN_USE_NEW_SUPPLIERMOD')) || (isModEnabled("supplier_order") && $user->hasRight('supplier_order', 'lire')) || (isModEnabled("supplier_invoice") && $user->hasRight('supplier_invoice', 'lire'))) {
 							$htmltext = $product_fourn->display_price_product_fournisseur(1, 1, 0, 1);
-							print '<span class="amount">'.$form->textwithpicto(price($product_fourn->fourn_unitprice * (1 - $product_fourn->fourn_remise_percent / 100) - $product_fourn->fourn_remise).' '.$langs->trans("HT"), $htmltext).'</span>';
+// VSPR fix: this should not be hardcoded // price should be created by function so we changed the function too. 2 now comes back with the number only
```

Comment:
Do not mix fix and adding new feature in same pr.

Reply:
ok, that was also a new one because the value now comes from the function and the function reacts to value showunitprice "2" it its call. I use that one in my hook to calculate the 5 discounts.